src/main/java/org/graphqlcrudjava/DatabaseAnalyzer.java

```diff
@@ -0,0 +1,158 @@
+package org.graphqlcrudjava;
+
+import org.graphqlcrudjava.model.Attribute;
+import org.graphqlcrudjava.model.Cardinality;
+import org.graphqlcrudjava.model.Entity;
+import org.graphqlcrudjava.model.Relation;
+import org.graphqlcrudjava.types.TypeMapImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class DatabaseAnalyzer implements AutoCloseable {
```

Comment:
I will build tendency to add javadoc early even for prototype as we see often prototypes being put into production with no docs :)